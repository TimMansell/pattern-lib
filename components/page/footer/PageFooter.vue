<template>
  <footer id="page-footer" class="page-footer">
    <PageFooterUpper />
    <div class="page-footer__bottom">
      <div class="page-footer__inner max">
        <div class="grid">
          <div class="cell cell--wide-3of4">
            <ToggleBlockMobile>
              <template slot="default">
                <h2>Site Map</h2>
              </template>
              <template slot="hidden">
                <PageFooterSitemap />
              </template>
            </ToggleBlockMobile>
          </div>
          <div class="page-footer__main__ack cell cell--wide-1of4">
            <p class="page-footer__ack">We acknowledge and pay respect to the Traditional Owners of the lands upon which our campuses are situated.</p>
          </div>
        </div>
      </div>
    </div>
  </footer>
</template>

<script>
import ToggleBlockMobile from '../../toggle/ToggleBlockMobile.vue';
import PageFooterUpper from './PageFooterUpper.vue';
import PageFooterSitemap from './PageFooterSitemap.vue';

export default {
  components: {
    ToggleBlockMobile,
    PageFooterUpper,
    PageFooterSitemap,
  },
};
</script>

