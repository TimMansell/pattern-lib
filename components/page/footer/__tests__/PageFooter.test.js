import { mount } from 'vue-test-utils';
import PageFooter from '../PageFooter.vue';

describe('PageFooter', () => {
  it('should match snapshot', () => {
    expect(mount(PageFooter).element).toMatchSnapshot();
  });
});
