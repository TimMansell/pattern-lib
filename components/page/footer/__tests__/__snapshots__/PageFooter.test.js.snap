// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter should match snapshot 1`] = `
<footer
  class="page-footer"
  id="page-footer"
>
  <div
    class="page-footer__top"
  >
    <div
      class="page-footer__inner max"
    >
      <div
        class="page-footer__header"
      >
        <div
          class="page-footer__header__logo"
        >
          <a
            class="link-img"
            href="https://www.unimelb.edu.au"
          >
            <img
              alt="The University of Melbourne homepage"
              height="117"
              src="../../shared/logo.svg"
              width="117"
            />
          </a>
        </div>
         
        <div
          class="page-footer__header__search"
        >
          <form
            action="https://search.unimelb.edu.au"
            aria-hidden="true"
            class="page-footer-search"
            id="search"
          >
            <div
              class="page-footer-search__field"
            >
              <label
                class="page-footer-search__label"
                for="f_s_q"
              >
                Search
              </label>
               
              <input
                aria-label="Search the University"
                class="page-footer-search__input"
                id="f_s_q"
                name="q"
                placeholder="Type to search"
                type="search"
              />
               
              <button
                aria-label="Submit"
                class="page-footer-search__submit"
                href="#"
                onclick="submit()"
                type="submit"
              >
                <svg
                  class="page-footer-search__icon"
                  focusable="false"
                  role="presentation"
                >
                  <use
                    xlink:href="#icon-search"
                  />
                </svg>
              </button>
            </div>
          </form>
        </div>
      </div>
       
      <div
        class="page-footer__main grid"
      >
        <div
          class="page-footer__main__users cell cell--wide-1of4"
        >
          <ul
            class="page-footer__users list-reset list-chevrons links-reset"
          >
            <li>
              <a
                href="https://students.unimelb.edu.au/"
              >
                Current students
              </a>
            </li>
             
            <li>
              <a
                href="https://staff.unimelb.edu.au/"
              >
                Staff
              </a>
            </li>
             
            <li>
              <a
                href="https://www.alumni.unimelb.edu.au/"
              >
                Alumni
              </a>
            </li>
          </ul>
        </div>
         
        <div
          class="page-footer__main__popular cell cell--wide-1of2"
        >
          <ul
            class="page-footer__popular list-reset links-reset"
          >
            <li>
              <a
                class="link-icon"
                href="https://about.unimelb.edu.au/strategy/our-structure/faculties-and-graduate-schools"
              >
                <svg
                  class="link-icon__icon svg"
                  focusable="false"
                  role="presentation"
                >
                  <use
                    xlink:href="#icon-faculty"
                  />
                </svg>
                <span
                  class="link-icon__text"
                >
                  Faculties & graduate schools
                </span>
              </a>
            </li>
             
            <li>
              <a
                class="link-icon"
                href="https://maps.unimelb.edu.au/"
              >
                <svg
                  class="link-icon__icon svg"
                  focusable="false"
                  role="presentation"
                >
                  <use
                    xlink:href="#icon-location"
                  />
                </svg>
                <span
                  class="link-icon__text"
                >
                  Maps
                </span>
              </a>
            </li>
             
            <li>
              <a
                class="link-icon"
                href="https://library.unimelb.edu.au/"
              >
                <svg
                  class="link-icon__icon svg"
                  focusable="false"
                  role="presentation"
                >
                  <use
                    xlink:href="#icon-library"
                  />
                </svg>
                <span
                  class="link-icon__text"
                >
                  Library
                </span>
              </a>
            </li>
             
            <li>
              <a
                class="link-icon"
                href="https://www.alumni.unimelb.edu.au/give"
              >
                <svg
                  class="link-icon__icon svg"
                  focusable="false"
                  role="presentation"
                >
                  <use
                    xlink:href="#icon-campaign"
                  />
                </svg>
                <span
                  class="link-icon__text"
                >
                  Support the campaign
                </span>
              </a>
            </li>
             
            <li>
              <a
                class="link-icon"
                href="https://www.unimelb.edu.au/contact/"
              >
                <svg
                  class="link-icon__icon svg"
                  focusable="false"
                  role="presentation"
                >
                  <use
                    xlink:href="#icon-phone"
                  />
                </svg>
                <span
                  class="link-icon__text"
                >
                  Contact us
                </span>
              </a>
            </li>
             
            <li>
              <a
                class="link-icon"
                href="https://about.unimelb.edu.au/careers"
              >
                <svg
                  class="link-icon__icon svg"
                  focusable="false"
                  role="presentation"
                >
                  <use
                    xlink:href="#icon-jobs"
                  />
                </svg>
                <span
                  class="link-icon__text"
                >
                  Jobs
                </span>
              </a>
            </li>
          </ul>
        </div>
         
        <div
          class="page-footer__main__contacts cell cell--wide-1of4"
        >
          <p
            class="screenreaders-only"
          >
            Follow us on:
          </p>
           
          <ul
            class="social-list list-reset links-reset page-footer__social"
          >
            <li
              class=""
            >
              <a
                class="social-list__link"
                href="https://www.facebook.com/unimelb"
              >
                <svg
                  aria-label="Facebook"
                  class="social-list__icon"
                  focusable="false"
                  height="22"
                  role="presentation"
                  width="22"
                >
                  <use
                    xlink:href="#icon-facebook"
                  />
                </svg>
                 
                <span
                  class="screenreaders-only"
                >
                  Facebook
                </span>
              </a>
            </li>
             
            <li
              class=""
            >
              <a
                class="social-list__link"
                href="https://twitter.com/unimelb"
              >
                <svg
                  aria-label="Twitter"
                  class="social-list__icon"
                  focusable="false"
                  height="22"
                  role="presentation"
                  width="22"
                >
                  <use
                    xlink:href="#icon-twitter"
                  />
                </svg>
                 
                <span
                  class="screenreaders-only"
                >
                  Twitter
                </span>
              </a>
            </li>
             
            <li
              class=""
            >
              <a
                class="social-list__link"
                href="https://www.linkedin.com/school/university-of-melbourne"
              >
                <svg
                  aria-label="LinkedIn"
                  class="social-list__icon"
                  focusable="false"
                  height="22"
                  role="presentation"
                  width="22"
                >
                  <use
                    xlink:href="#icon-linkedin"
                  />
                </svg>
                 
                <span
                  class="screenreaders-only"
                >
                  LinkedIn
                </span>
              </a>
            </li>
          </ul>
           
          <dl
            class="page-footer__contact list-def list-def--wrap"
          >
            <dt>
              Phone:
            </dt>
             
            <dd>
              13 MELB 
              <small>
                (
                <a
                  href="tel:136352"
                >
                  13 6352
                </a>
                )
              </small>
            </dd>
             
            <dt>
              International:
            </dt>
             
            <dd>
              <a
                href="tel:+61390355511"
              >
                +61 3 9035 5511
              </a>
            </dd>
             
            <dt>
              ABN:
            </dt>
             
            <dd>
              84 002 705 224
            </dd>
             
            <dt>
              CRICOS Provider Code:
            </dt>
             
            <dd>
              00116K 
              <small>
                (
                <a
                  href="https://services.unimelb.edu.au/international/visas"
                >
                  visa information
                </a>
                )
              </small>
            </dd>
          </dl>
        </div>
      </div>
       
      <div
        class="page-footer__legal links-reset"
      >
        <a
          href="https://safety.unimelb.edu.au/emergency"
        >
          Emergency information
        </a>
         
        <span
          class="page-footer__legal__sep"
        >
          |
        </span>
         
        <a
          href="https://www.unimelb.edu.au/governance/disclaimer"
        >
          Disclaimer & copyright
        </a>
         
        <span
          class="page-footer__legal__sep"
        >
          |
        </span>
         
        <a
          href="https://www.unimelb.edu.au/accessibility"
        >
          Accessibility
        </a>
         
        <span
          class="page-footer__legal__sep"
        >
          |
        </span>
         
        <a
          href="https://www.unimelb.edu.au/governance/compliance/privacy"
        >
          Privacy
        </a>
      </div>
    </div>
  </div>
   
  <div
    class="page-footer__bottom"
  >
    <div
      class="page-footer__inner max"
    >
      <div
        class="grid"
      >
        <div
          class="cell cell--wide-3of4"
        >
          <div
            class="toggleblock--mobile toggleblock"
          >
            <div
              aria-controls="ui-toggle-14-panel-1"
              class="toggleblock__default"
              id="ui-toggle-14-header-1"
              tabindex="0"
            >
              <h2>
                Site Map
              </h2>
                
            </div>
             
            <div
              aria-hidden="true"
              aria-labelledby="ui-toggle-14-header-1"
              class="toggleblock__hidden"
              id="ui-toggle-14-panel-1"
              role="region"
              tabindex="-1"
            >
              <div
                class="grid"
              >
                <div
                  class="cell cell--tab-1of2 cell--desk-1of3"
                >
                  <a
                    class="page-footer__sitemap-heading"
                    href="https://study.unimelb.edu.au/"
                    tabindex="-1"
                  >
                    Study at Melbourne
                  </a>
                   
                  <ul
                    class="page-footer__sitemap-pages page-footer__sitemap-pages--minor list-reset list-chevrons"
                  >
                    <li>
                      <a
                        class="link-reset"
                        href="https://study.unimelb.edu.au/find"
                        tabindex="-1"
                      >
                        Find a course
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://study.unimelb.edu.au/how-to-apply"
                        tabindex="-1"
                      >
                        Admissions, fees & applications
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://study.unimelb.edu.au/connect-with-us/information-for-schools/information-for-schools-outside-australia"
                        tabindex="-1"
                      >
                        International students
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://www.unimelb.edu.au/campustour"
                        tabindex="-1"
                      >
                        Campus tour
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://study.unimelb.edu.au/connect-with-us/"
                        tabindex="-1"
                      >
                        Connect with us
                      </a>
                    </li>
                  </ul>
                </div>
                 
                <div
                  class="cell cell--tab-1of2 cell--desk-1of3"
                >
                  <a
                    class="page-footer__sitemap-heading"
                    href="https://about.unimelb.edu.au/"
                    tabindex="-1"
                  >
                    About us
                  </a>
                   
                  <ul
                    class="page-footer__sitemap-pages page-footer__sitemap-pages--minor list-reset list-chevrons"
                  >
                    <li>
                      <a
                        class="link-reset"
                        href="https://about.unimelb.edu.au/strategy-and-leadership"
                        tabindex="-1"
                      >
                        Strategy and leadership
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://about.unimelb.edu.au/facts-and-figures"
                        tabindex="-1"
                      >
                        Tradition of excellence
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://iro.unimelb.edu.au/"
                        tabindex="-1"
                      >
                        International connections
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://maps.unimelb.edu.au"
                        tabindex="-1"
                      >
                        Campuses and facilities
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://about.unimelb.edu.au/strategy/our-structure"
                        tabindex="-1"
                      >
                        Structure and governance
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://about.unimelb.edu.au/news-resources/reports-and-publications"
                        tabindex="-1"
                      >
                        Policy and publications
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://about.unimelb.edu.au/careers"
                        tabindex="-1"
                      >
                        Careers at Melbourne
                      </a>
                    </li>
                  </ul>
                </div>
                 
                <div
                  class="cell cell--tab-1of2 cell--desk-1of3 cell--tab-clear"
                >
                  <a
                    class="page-footer__sitemap-heading"
                    href="https://www.unimelb.edu.au/contact"
                    tabindex="-1"
                  >
                    Contact & Maps
                  </a>
                   
                  <ul
                    class="page-footer__sitemap-pages page-footer__sitemap-pages--minor list-reset list-chevrons"
                  >
                    <li>
                      <a
                        class="link-reset"
                        href="https://ask.unimelb.edu.au/app/ask"
                        tabindex="-1"
                      >
                        Enquiries
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        data-notsecure=""
                        href="http://newsroom.melbourne.edu/"
                        tabindex="-1"
                      >
                        Media
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://findanexpert.unimelb.edu.au/"
                        tabindex="-1"
                      >
                        Find an expert
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://maps.unimelb.edu.au/"
                        tabindex="-1"
                      >
                        Campus maps
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://about.unimelb.edu.au/news-resources/campus-services-and-facilities/transport-and-parking"
                        tabindex="-1"
                      >
                        Traffic, parking & bicycles
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        data-notsecure=""
                        href="http://directory.unimelb.edu.au/"
                        tabindex="-1"
                      >
                        Find a staff member
                      </a>
                    </li>
                  </ul>
                </div>
                 
                <div
                  class="cell cell--tab-1of2 cell--desk-1of3 cell--desk-clear cell--wide-clear"
                >
                  <a
                    class="page-footer__sitemap-heading"
                    href="https://www.alumni.unimelb.edu.au/"
                    tabindex="-1"
                  >
                    Alumni & friends
                  </a>
                   
                  <ul
                    class="page-footer__sitemap-pages page-footer__sitemap-pages--minor list-reset list-chevrons"
                  >
                    <li>
                      <a
                        class="link-reset"
                        href="https://www.alumni.unimelb.edu.au/benefits"
                        tabindex="-1"
                      >
                        Benefits & services
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://www.alumni.unimelb.edu.au/give/"
                        tabindex="-1"
                      >
                        Give
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://www.alumni.unimelb.edu.au/volunteer"
                        tabindex="-1"
                      >
                        Volunteer
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://mag.alumni.unimelb.edu.au/?sl=1"
                        tabindex="-1"
                      >
                        3010: alumni magazine
                      </a>
                    </li>
                  </ul>
                </div>
                 
                <div
                  class="cell cell--tab-1of2 cell--desk-1of3 cell--tab-clear"
                >
                  <a
                    class="page-footer__sitemap-heading"
                    href="https://research.unimelb.edu.au/"
                    tabindex="-1"
                  >
                    Research
                  </a>
                   
                  <ul
                    class="page-footer__sitemap-pages page-footer__sitemap-pages--minor list-reset list-chevrons"
                  >
                    <li>
                      <a
                        class="link-reset"
                        href="https://research.unimelb.edu.au/melbourne-research-institutes"
                        tabindex="-1"
                      >
                        Research institutes
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://findanexpert.unimelb.edu.au/"
                        tabindex="-1"
                      >
                        Find an expert or supervisor
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://gradresearch.unimelb.edu.au/"
                        tabindex="-1"
                      >
                        Graduate researchers
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://pursuit.unimelb.edu.au/"
                        tabindex="-1"
                      >
                        Pursuit: our research showcase
                      </a>
                    </li>
                  </ul>
                </div>
                 
                <div
                  class="cell cell--tab-1of2 cell--desk-1of3"
                >
                  <a
                    class="page-footer__sitemap-heading"
                    href="https://engagement.unimelb.edu.au/"
                    tabindex="-1"
                  >
                    Engagement
                  </a>
                   
                  <ul
                    class="page-footer__sitemap-pages page-footer__sitemap-pages--minor list-reset list-chevrons"
                  >
                    <li>
                      <a
                        class="link-reset"
                        href="https://events.unimelb.edu.au/"
                        tabindex="-1"
                      >
                        Events
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        data-notsecure=""
                        href="http://www.sport.unimelb.edu.au/fitnesscentres"
                        tabindex="-1"
                      >
                        Sports facilities
                      </a>
                    </li>
                     
                    <li>
                      <a
                        class="link-reset"
                        href="https://shop.unimelb.edu.au/"
                        tabindex="-1"
                      >
                        Shop
                      </a>
                    </li>
                  </ul>
                </div>
              </div>
               
              <a
                class="toggleblock__footer"
              >
                Close panel
              </a>
            </div>
          </div>
        </div>
         
        <div
          class="page-footer__main__ack cell cell--wide-1of4"
        >
          <p
            class="page-footer__ack"
          >
            We acknowledge and pay respect to the Traditional Owners of the lands upon which our campuses are situated.
          </p>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
